eng/targets/GenerateServiceHubConfigurationFiles.targets

```diff
@@ -11,14 +11,15 @@
 
   <Target Name="CalculateServiceHubConfigurationFiles">
     <ItemGroup>
-      <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="" HostSuffix=".x86" HostIdSuffix="32" />
-      <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="64" HostSuffix="" HostIdSuffix="" />
-      <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="64S" HostSuffix="" HostIdSuffix="S" />
+      <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="Desktop" HostSuffix=".x86" HostIdSuffix="32" RuntimePrefix="desktop"/>
```

Comment:
📝 I'll send a separate PR to remove this one

Reply:
I can just remove it as part of this, to avoid conflict

Reply:
I'd prefer if we remove the desktop as well. We should just have on - Core.

Reply:
desktop will need to stay for a while, we are going to run servicehub core in an experiment first.

Reply:
That does not help our analyzer/code generator story. Also adds another 50 files or so of service.json files.

Reply:
OK, clarified offline. We'll keep desktop temporarily.

Reply:
Support for x86 host is removed